apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -222,6 +230,40 @@ public override int ExecuteNonQuery() => ExecuteNonQueryAsync(_synchronousCancel
                     "You can only call ExecuteNonQueryAsync on a Delete, Insert, InsertUpdate, or Update Command");
             }
 
+            if (SpannerCommandTextBuilder.SpannerCommandType == SpannerCommandType.Ddl)
+            {
+                return ExecuteDdlAsync(cancellationToken);
+            }
+            else
+            {
+                return ExecuteMutationsAsync(cancellationToken);
+            }
+        }
+
+        private async Task<int> ExecuteDdlAsync(CancellationToken cancellationToken)
+        {
+            var databaseAdminClient = await DatabaseAdminClient.CreateAsync();
```

Comment:
ConfigureAwait (and below).
We should sync up on when we're going to merge all this into master so we can get the automated checking in...